subprojects/packagefiles/mpdecimal/tests++/meson.build

```diff
@@ -0,0 +1,27 @@
+conf = configuration_data()
+
+if cc.has_header('pthread.h')
+  conf.set('HAVE_PTHREAD_H', 1)
+endif
+
+configure_file(
+  output: 'config.h',
+  configuration: conf)
+
+threads_dep = dependency('threads')
+
+runtest = executable('runtest', files('runtest.cc', 'test.cc'), dependencies: [mpdecpp_dep, threads_dep])
+
+# These tests require to download test data.
+# test(
+#   'libmpdec++/official', runtest,
+#   args: files('official.topTest'),
+#   depends: testdata,
+#   workdir: join_paths(meson.current_build_dir(), '..', 'tests'))
+
+test(
+  'libmpdec++/additional', runtest,
+  args: files('additional.topTest'),
```

Comment:
(Although this does need to be files() because it is added as string arguments to a test, and hence needs to remember the directory it came from.)

Reply:
This needs to be `files()` otherwise it does not work, because if the `workdir` parameter, I think.

Reply:
Yes, I did say that :) to clarify that my above comment doesn't apply here.